pmd-julia/src/main/antlr4/net/sourceforge/pmd/lang/julia/ast/Julia.g4

```diff
@@ -0,0 +1,163 @@
+grammar Julia;
```

Comment:
Did you create this grammar yourself? Have you taken a look at [this one](https://github.com/JuliaEditorSupport/JuliaDT/blob/master/plugins/com.juliacomputing.jldt.eclipse.parser/src/main/antlr4/org/julia/lang/parser/Julia.g4)? It looks more complete, albeit unmaintained

Reply:
This grammar was made by Paul Jansen from TIOBE. In any case, I'll have a look at the grammar you suggested.

Reply:
We decided that it is indeed better to take your suggestion as a starting point. It will require some additional work though.

Reply:
I'd vote for using this grammar for now. We can change it always later on. Right now, it's only used for CPD/tokenization, so there should be no compatibility issues when we change the grammar later.